# Describe file
k_p_trans: 10.0

k_p_rot: 10.0

k_i_trans: 0.1

k_i_rot: 0.001

linear_threshold: 0.01 # Describe

angular_threshold: 0.001 # Describe

